viewer-prototype/src/node/trace-server-service.ts

```diff
@@ -4,11 +4,24 @@ import { TraceServerConfigService } from '../common/trace-server-config';
 @injectable()
 export class TraceServerServiceImpl implements TraceServerConfigService {
 
-    async startTraceServer(path: string | undefined, port: number | undefined): Promise<void> {
-        spawn(`${path}`, ['-vmargs', `-Dtraceserver.port=${port}`]);
+    startTraceServer(path: string | undefined, port: number | undefined): Promise<void> {
+        const server = spawn(`${path}`, ['-vmargs', `-Dtraceserver.port=${port}`]);
```

Comment:
I'm not suggesting to merge the following lines, but during debugging it can be useful because you'll see what the server outputs to the console:
         server.stdout.on('data', data => {
            console.log(\`stdout:\n${data}\`);
          });
          server.stderr.on('data', data => {
            console.error(\`stderr: ${data}\`);
          });